app/config.js

```diff
@@ -4,11 +4,11 @@ var oauth = require('../oauth.json');
 module.exports = {
   api: process.env.API_URL || oauth.api || 'https://api.github.com',
   apiStatus: process.env.API_STATUS_URL || oauth.status || 'https://status.github.com/api/status.json',
-  site: process.env.OAUTH_URL || oauth.site || 'https://github.com',
+  host: process.env.OAUTH_HOST || oauth.host || 'github.com',
```

Comment:
@jmatsushita heads up, this will be a slight breaking change for your next deployment. Instead of `https://code.iilab.org` it'll just be `code.iilab.org`. This lets us do better url string creation using node's `path` and `url` modules.